### ISSUE TITLE
.return is not a function

### ISSUE THREAD

**Issue by DanielWFrancis** [arabold/serverless-export-env#26]
Builds started failing on my export-env call with this

```
Type Error ---------------------------------------------
 
  TypeError: AWS.request(...).then(...).return is not a function
      at describeStack (/home/circleci/.../node_modules/serverless-export-env/src/lib/collectStackOutputs.js:16:12)
```

I don't actually write much Javascript but I went through removing the . for the above instance and two more instances that threw errors:

```
TypeError: AWS.request(...).then(...).return is not a function
      at listStackResources (/home/circleci/.../node_modules/serverless-export-env/src/lib/resolveCloudFormationEnvVariables.js:32:12)
```

```
TypeError: AWS.request(...).tap is not a function
      at listExports (/home/circleci/.../node_modules/serverless-export-env/src/lib/resolveCloudFormationEnvVariables.js:9:6)
```

Not sure if this solution creates race conditions etc., but it ran fine just now. Also curious about what changed to cause this to not work?

Thank you :)

**Comment by pdecarcer** [comment]
I run on a mac just fine, but when I try tu run on a ubuntu get 
`TypeError: AWS.request(...).then(...).return is not a function` Error

**Comment by ivanamandic** [comment]
Hi, I just updated serverless framework and now `serverless export-env -s develop` not working any more. 
I get `AWS.request(...).then(...).return is not a function` error.

**Comment by coyoteecd** [comment]
Also ran into this today after updating to serverless v2.23. 
I think this is related to these refactors: https://github.com/serverless/serverless/pull/8871/commits/b5a70fbdbbbf9145ce210c562ffac666d32e7d7b

Serverless Framework is slowly deprecating BbPromise and replacing it with native async/await. This plugin, and possibly others, uses BbPromise and also assumes that async operations from serverless are BbPromise instances. Not anymore, it seems.

**Comment by pdecarcer** [comment]
@coyoteecd So you propose to downgrade serverless before v2.23??

**Comment by DanielWFrancis** [comment]
@coyoteecd interesting. Does seem relevant, but rolling back to v2.20.1 still is returning the error oddly enough.

From my last run, which ended with the same error as in the op:
```
  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              10.19.0
     Framework Version:         2.20.1
     Plugin Version:            4.4.2
     SDK Version:               2.3.2
     Components Version:        3.6.2
```

**Comment by pdecarcer** [comment]
I just installed version 2.22.0 and worked with out a problem!

**Comment by gerdzhikov** [comment]
Same problem here after upgrading to serverless 2.27.0 running it on OSX:

```
 Your Environment Information ---------------------------
     Operating System:          darwin
     Node Version:                  14.16.0
     Framework Version:        2.27.0 (local)
     Plugin Version:                4.4.3
     SDK Version:                   2.3.2
     Components Version:     3.7.1

```

Can confirm, that it works for me fine with 2.22.0

**Comment by williamsandonz** [comment]
Would be great to get this one solved

**Comment by williamsandonz** [comment]
Hey everyone, I've taken a look into this and the issue was actually fixed in the github repo [here](https://github.com/arabold/serverless-export-env/pull/28/files) but a new npm package hasn't been published, so if you need a temp work around just npm install arabold/serverless-export-env to get the changes, then can switch back to NPM package once released. @arabold  you able to publish a new version so that everyone can get their updates? Would also appreicate if you could check out my PR (unrelated) about the overwrite flag.

**Comment by pgrzesik** [comment]
Hello @arabold :wave: As @williamsandonz mentioned above, the fix has been merged, but the new version wasn't released to `npm`. Would you be able to release a new version? If you're no longer interested in maintaining the plugin, I'm happy to jump in and help out with such maintenance - please let me know if that sounds good :bow:

**Comment by nthornton2010** [comment]
@arabold any update on when this fix will make it to npm?

**Comment by huntedman** [comment]
As a temporary workaround you can do:

npm i -D https://github.com/arabold/serverless-export-env.git

It installs the dependency straight from github, skipping the NPM.

**Comment by arabold** [comment]
Fixed in v1.4.2. Sorry for the delay 😞
